modules/sharethroughBidAdapter.js

```diff
@@ -147,6 +135,34 @@ export const sharethroughAdapterSpec = {
   onSetTargeting: (bid) => {}
 };
 
+function handleUniversalIds(bidRequest) {
+  if (!bidRequest.userId) return {};
+
+  const universalIds = {};
+
+  const ttd = utils.deepAccess(bidRequest, 'userId.tdid');
+  if (ttd) universalIds.ttduid = ttd;
+
+  const pubc = utils.deepAccess(bidRequest, 'userId.pubcid') || utils.deepAccess(bidRequest, 'crumbs.pubcid');
+  if (pubc) universalIds.pubcid = pubc;
+
+  const idl = utils.deepAccess(bidRequest, 'userId.idl_env');
+  if (idl) universalIds.idluid = idl;
+
+  const id5 = utils.deepAccess(bidRequest, 'userId.id5id');
+  if (id5) universalIds.id5uid = id5;
```

Comment:
just as an fyi, the `userId.id5id` is an object, not a string. the object looks like this:
```javascript
{
  uid: 'ID5-12345', // this is the ID5 Universal ID
  ext: {
    linkType: 2 // possible options are 0, 1, or 2
  }
}
```

It is highly recommended that you pass through the entire object to your servers so you can pass through the `linkType` to DSPs, several of whom have asked for this field. If you have any questions, feel free to reach out to me directly (scott at id5 dot io)

Reply:
@smenzer thanks for raising that, we missed it and I'm gonna push a fix soon. Is there a doc somewhere I can link internally to describe the meaning of `linkType` and its possible values?

Reply:
@maphe great to hear! we have some docs on our wiki - https://wiki.id5.io/x/EABH - but basically, `linkType` can be 0 (not linked across domains), 1 (linked across domains with probabilistic signals), or 2 (linked across domains with deterministic signals)

Reply:
> @smenzer thanks for the link, that's perfect. Also I addressed your comments 👍 feel free to take another look

Looks great!